build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
 
     dependencies {
-        classpath("io.spring.gradle:dependency-management-plugin:1.0.3.RELEASE")
+        classpath("io.spring.gradle:dependency-management-plugin:1.0.11.RELEASE")
```

Comment:
In my opinion, there is no need to update this version, this updates the library versions, and we need to retest all the functionality, and we only need to correct the publish to Maven.